src/number_theory/arithmetic_function.lean

```diff
@@ -732,6 +732,25 @@ begin
   { rcases h with h | h; simp [h] }
 end
 
+lemma is_multiplicative_moebius : is_multiplicative μ :=
+⟨by simp, begin
```

Comment:
With an extra lemma about multiplicative functions in general, this simplifies too:
```
lemma iff_ne_zero [monoid_with_zero R] (f : arithmetic_function R) :
  is_multiplicative f ↔
    f 1 = 1 ∧ (∀ {m n : ℕ}, m ≠ 0 → n ≠ 0 → m.coprime n → f (m * n) = f m * f n) :=
begin
  refine and_congr_right' (forall₂_congr (λ m n, ⟨λ h _ _, h, λ h hmn, _⟩)),
  rcases eq_or_ne m 0 with rfl | hm,
  { simp },
  rcases eq_or_ne n 0 with rfl | hn,
  { simp },
  exact h hm hn hmn,
end
```
then
```
lemma is_multiplicative_moebius : is_multiplicative μ :=
begin
  rw is_multiplicative.iff_ne_zero,
  refine ⟨by simp, λ n m hn hm hnm, _⟩,
  simp only [moebius, zero_hom.coe_mk, ←squarefree_mul hnm, ite_and, card_factors_mul hn hm],
  rw [pow_add, ite_mul_zero_right, ite_mul_zero_left],
end
```

Reply:
Thanks! Requires adding an import, but I assume that's fine. :-)

Reply:
Oops I should've mentioned that, sorry!